### PR TITLE
Added functionality for `grades_persistent_tables

### PR DESCRIPTION
**Pull request by lstmemery**
I've added functionality creating and inserting data into `grades_persistent` and `grades_persistent_subsection.`

The command to create `grades_persistent` is:
`grades_persistent`

to create `grades_persistent_subsection`:
`--subsection grades_persistent`.

I've added the schemas. The descriptions were populated from [here](http://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/sql_schema.html).

The main function I created is `make_grading_policy_table.upload_grade_persistent_data.` I have tested it. I have also added the two versions of this function in `doall`. 

If you need me to do anything else, I'll happily oblige.

### CODE REVIEW

**Comment by ichuang** [comment]
Thanks for the contribution!  The helpful comments are especially appreciated.

Some questions:

What's the role of the `edx2bigquery/make_grades _table.py` file?

Also, does it make sense to put the `upload_grades_persistent_data` function inside the `edx2bigquery/make_grading_policy_table.py` file, given that it is doing something separate from the `grading_policy` data?

File: edx2bigquery/main.py [hunk 1]
```diff
@@ -2133,7 +2174,11 @@ def CommandLine():
         run_parallel_or_serial(axis2bq, param, courses, args, parallel=args.parallel)
         # axis2bq(param, args, args)
 
-    elif (args.command=='grading_policy'):
```
Comment (ichuang) [inline edx2bigquery/main.py:2136]: Why did you remove the `grading_policy` command?

File: edx2bigquery/main.py [hunk 2]
```diff
@@ -968,15 +998,20 @@ def doall(param, course_id, args, stdout=None):
...
         person_day(param, course_id, args, stop_on_error=False)
         enrollment_day(param, course_id, args)
-	enrollment_events_table(param, course_id, args)
+    	enrollment_events_table(param, course_id, args)
         person_course(param, course_id, args)
         problem_check(param, course_id, args)
         show_answer_table(param, course_id, args)
         analyze_ora(param, course_id, args)
         time_on_task(param, course_id, args, just_do_totals=True, suppress_errors=True)
-        make_grading_policy(param, course_id, args)
```
Comment (ichuang) [inline edx2bigquery/main.py:977]: Why remove the call to `make_grading_policy` in doall?

**Review by ichuang — CHANGES_REQUESTED** [review]

**Comment by lstmemery** [comment]
I've re-added the `grading_policy` functionality (my mistakes!) and moved all the new functionality into `make_grades_persistent.py`.

File: edx2bigquery/main.py (hunk 1 above)
Comment (lstmemery) [reply edx2bigquery/main.py:2136]: Fixed!

File: edx2bigquery/main.py (hunk 2 above)
Comment (lstmemery) [reply edx2bigquery/main.py:977]: Fixed!

File: edx2bigquery/main.py [hunk 3]
```diff
@@ -17,6 +18,8 @@
 from argparse import RawTextHelpFormatter
 from collections import OrderedDict
 
+import edx2bigquery.make_grades_persistent
```
Comment (ichuang) [inline edx2bigquery/main.py]: Is this import statement needed?

Comment (lstmemery) [reply edx2bigquery/main.py]: Nope. It was an artifact of a refactoring.

**Review by ichuang — APPROVED** [review]

**Comment by ichuang** [comment]
Thanks for the helpful contribution!

**Comment by SdotGlenn** [comment]
Many thanks Matt for this!

**Pull request merged**